packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -618,6 +619,104 @@ class TextMapPropagator {
 
     return spanContext._traceId.toString(16)
   }
+
+  static _convertOtelContextToDatadog (traceId, spanId, traceFlag, ts, meta = {}) {
+    const origin = null
+    let samplingPriority = traceFlag
+
+    ts = ts?.traceparent || null
+
+    if (ts) {
+      const tsList = ts.split(',').map(member => member.trim())
+      ts = tsList.join(',')
+      if (/[^\x20-\x7E]+/.test(ts)) {
```

Comment:
We already have [a tracestate parser](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/opentracing/propagation/tracestate.js). Is there some reason not to just use that?